core/scripts/go.mod

```
@@ -1,6 +1,6 @@
 module github.com/smartcontractkit/chainlink/core/scripts
 
-go 1.21.3
+go 1.21.5
```

We're up to 1.21.7 now
```suggestion
go 1.21.7
```